### ISSUE TITLE
Does not offer to create .zshrc

### ISSUE THREAD

**Issue by deni** [maxgoedjen/secretive#186]
Upon setup, if `.zshrc` does not exist, one will have to create it manually from Terminal. After this, doing `touch ~/.zshrc` and then asking Secretive to populate the file, the file was still empty.

**Comment by maxgoedjen** [comment]
Yeah that's not a terribly well-handled case. Sandboxing might make it pretty annoying to address though :/
(not sure what's going on with the post-touch one though)
